apollo-gradle-plugin/src/main/groovy/com/apollographql/android/gradle/ApolloPlugin.groovy

```diff
@@ -60,54 +61,61 @@ class ApolloPlugin implements Plugin<Project> {
           dep.name == API_DEP_NAME
         }
         if (apolloApiDep != null && apolloApiDep.version != VersionKt.VERSION) {
-          throw new GradleException("apollo-api version ${apolloApiDep.version} isn't compatible with the apollo-gradle-plugin version ${VersionKt.VERSION}")
+          throw new GradleException(
+              "apollo-api version ${apolloApiDep.version} isn't compatible with the apollo-gradle-plugin version ${VersionKt.VERSION}")
         }
         if (System.getProperty("apollographql.skipApi") != "true" && apolloApiDep == null) {
           compileDepSet.add(project.dependencies.create("$APOLLO_GROUP:$API_DEP_NAME:$VersionKt.VERSION"))
         }
-        project.getGradle().removeListener(this)
       }
 
       @Override
-      void afterResolve(ResolvableDependencies resolvableDependencies) {}
+      void afterResolve(ResolvableDependencies resolvableDependencies) {
+        resolvableDependencies.getResolutionResult().allDependencies.each {
+          if (it.toString().contains("com.google.guava:guava")) {
+            hasGuava = true
```

Comment:
can we simplify: `hasGuava |=  it.toString().contains("com.google.guava:guava")`?